gatewayFunctions.cmake

```diff
@@ -113,7 +113,7 @@ function(findAndInstall libraryName version submoduleRootDirectory cmakeRootDire
             #If the library directory doesn't exist, pull submodules
             if(NOT EXISTS "${cmakeRootDirectory}/CMakeLists.txt")
                 execute_process(
-                    COMMAND git submodule update --init ${submoduleRootDirectory}
+                    COMMAND git submodule update --init --recursive ${submoduleRootDirectory}
```

Comment:
I don't think that's a new dependency. I recall that Andrew did something special in the unit test case.